data/core/macros/ai.cfg

```diff
@@ -375,6 +375,7 @@
 #enddef
 
 #define AI_SCALE_ATTACK_DEPTH_BY_DIFFICULTY
+    #deprecated 3 1.17 The attack_depth aspect has no effect on the AI any more.
```

Comment:
Maybe remove the contents of the macro, since they won't do anything anyway? Not strictly required, I suppose. Not doing so means that use of this macro would cause a failure of schema validation; not sure whether that's good or bad though.

Reply:
Yeah, good point.  There's no need to have both a deprecation message and a schema error.  Done: c0bc8c5